### DISCUSSION TITLE
Can't get the setup file to actually run

### DISCUSSION THREAD

**Discussion by Tazzetiel** [joncage/ed-scout#200] [Q&A]
When I execute the Setup file, all that happens is that it opens a UAC window. When I confirm that, nothing more happens whatsoever.

What could be my issue, and what information do you know to answer the first part of the question?

**Comment by nmgsimulations** [comment]
See this post: [https://github.com/joncage/ed-scout/discussions/202](https://github.com/joncage/ed-scout/discussions/202)
I have attached an old version of EDScout in a ZIP file that I still use. Not the latest, but it works!

**Reply by Trickyhicky** [reply]
This solution has worked for me thanks to @nmgsimulations guys
https://github.com/joncage/ed-scout/discussions/202

**Reply by joncage** [reply]
Should be fixed in 1.9.0 - sorry about the wait ;)
